### PR TITLE
{lib}[GCCcore/11.3.0] libyaml v0.2.5, PyYAML v6.0 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by casparvl**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by casparvl** [comment]
@boegelbot please test @ generoso

**Comment by casparvl** [comment]
Test report by @casparvl
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
tcn454 - Linux RHEL 8.4, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/76182fb9f7bc6bdd2612c9e4858ebd0f for a full test report.

**Comment by casparvl** [comment]
Test report by @casparvl
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
software1.lisa.surfsara.nl - Linux debian 10.12, x86_64, Intel(R) Xeon(R) Silver 4110 CPU @ 2.10GHz, Python 3.7.3
See https://gist.github.com/919490c47a8e1ff0f0e53cb905c8dfa2 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/da200fb61ce1e76ecf260c7065ea62df for a full test report.

File: easybuild/easyconfigs/l/libyaml/libyaml-0.2.5-GCCcore-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+sources = ['yaml-%(version)s.tar.gz']
+checksums = ['c642ae9b75fee120b2d96c712538bd2cf283228d2337df2cf2988e3c02678ef4']
+
+builddependencies = [
+    ('binutils', '2.38'),
+]
+
+
+sanity_check_paths = {
+    'files': ['include/yaml.h', 'lib/%(name)s.a', 'lib/%(name)s.so'],
```
Comment (verdurin) [inline easybuild/easyconfigs/l/libyaml/libyaml-0.2.5-GCCcore-11.3.0.eb:21]: @casparvl please restore the handling of shared library file extensions with `SHLIB_EXT`.

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/l/libyaml/libyaml-0.2.5-GCCcore-11.3.0.eb (hunk 1 above)
Comment (casparvl) [reply easybuild/easyconfigs/l/libyaml/libyaml-0.2.5-GCCcore-11.3.0.eb]: Done. Silly `--try-toolchain-version`  seems to have resolved it... :)

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/f506ff6b037f96e62cbfdc9569e6f7df for a full test report.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @casparvl!

**Pull request merged**